lib/Controller/RoomController.php

```diff
@@ -441,6 +441,8 @@ protected function formatRoomV2(Room $room, ?Participant $currentParticipant): a
 			'canLeaveConversation' => false,
 			'canDeleteConversation' => false,
 			'notificationLevel' => Participant::NOTIFY_NEVER,
+			'lobbyState' => Webinary::LOBBY_NONE,
+			'lobbyTimer' => 0,
```

Comment:
Should this be added in the default case of `formatRoomV1` too?

Reply:
![https://i.ytimg.com/vi/vYeKsFEiSi8/hqdefault.jpg](https://i.ytimg.com/vi/vYeKsFEiSi8/hqdefault.jpg)